index.js

```diff
@@ -138,47 +185,69 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
+const ticketNum = []
+
+function takeATicketNumber(line){
+   
+  let num = ticketNum.length + 1
+  line.push(num)
+  ticketNum.push(num)
+  return `Welcome. You are ticket number ${num}`
+}
```

Comment:
Nice approach! This can be simplified. You're using an array to store the ticket numbers that have been used, but you're never actually using any of those values. Instead, you're using the length of that array. What if you just use a variable holding a Number data type to keep track of which ticket number  you are currently on instead?